.shippable.yml

```diff
@@ -4,8 +4,7 @@ compiler: gcc
 
 env:
     global:
-        - SDK=0.10.1
```

Comment:
Any reason why you've removed this?

Reply:
@galak ^^

Reply:
Didn't seem like its used as far as I can tell.